public/js/cita.json

```diff
@@ -5008,5 +4868,305 @@
     "lng": -2.5934541,
     "id": "94e94f50-1d30-49f0-8959-b2a799ba3423",
     "booking_location_id": "b74377da-d06b-48d0-9d74-cc362bbb53b3"
+  },
+  {
+    "title": "Poyton Civic Hall",
+    "address": "Park Lane\nCheshire East\nPoyton\nSK12 1GP",
+    "phone": null,
+    "hours": null,
+    "lat": 53.3494644,
+    "lng": -2.1180696,
+    "id": "dc4e84b0-ab6d-44a3-8090-d633b5ed5313",
+    "booking_location_id": "df38d9b9-e9a9-4bf0-b56b-4309965b815c"
+  },
+  {
+    "title": "Peterlee Macmillan Information Centre",
+    "address": "20 The Upper Chare\nPeterlee\nSR8 1BW",
+    "phone": null,
+    "hours": null,
+    "lat": 54.7589267,
+    "lng": -1.3343109,
+    "id": "5b061a57-4095-499e-a63b-26f4585a03ea",
+    "booking_location_id": "ee399dec-f9c2-4201-b27f-e3da114d3e17"
+  },
+  {
+    "title": "Bishop Auckland Hospital",
+    "address": "Cockton Hill Road\nBishop Auckland\nDurham\nDL14 6AD",
+    "phone": null,
+    "hours": null,
+    "lat": 54.6532811,
+    "lng": -1.6778693,
+    "id": "3e5f4f69-56cc-4ff1-bda5-d61ef1592693",
+    "booking_location_id": "ee399dec-f9c2-4201-b27f-e3da114d3e17"
+  },
+  {
+    "title": "University Hospital North Durham Hospital",
+    "address": "North Road\nDurham\nDH1 5TW",
+    "phone": null,
+    "hours": null,
+    "lat": 54.7825894,
+    "lng": -1.5859521,
+    "id": "bab77727-6757-4396-84e3-0d49dc0aba01",
+    "booking_location_id": "ee399dec-f9c2-4201-b27f-e3da114d3e17"
+  },
+  {
+    "title": "Stockport Library",
+    "address": "Wellington Rd South\nStockport\nSK1 3RS",
+    "phone": null,
+    "hours": null,
+    "lat": 53.4079067,
+    "lng": -2.1607592,
+    "id": "d7d58f46-acd6-4379-852f-dc426834ac2c",
+    "booking_location_id": "38fe9633-d556-4e96-ab99-518eeaf7d2a7"
+  },
+  {
+    "title": "Blackburn Library",
+    "address": "Town Hall Street\nBlackburn\nLancashire\nBB2 1AG",
+    "phone": null,
+    "hours": null,
+    "lat": 53.74968149999999,
+    "lng": -2.4850267,
+    "id": "a1e0209b-4585-4b26-9a30-aad795424dce",
+    "booking_location_id": "c554946e-7b79-4446-b2cd-d930f668e54b"
+  },
+  {
+    "title": "Rossendale",
+    "address": "Stublee Hall\nStublee Park\nBacup\nLancashire\nOL13 ODE",
+    "phone": null,
+    "hours": null,
+    "lat": null,
```

Comment:
Good spot! [Close to the edge](https://www.google.co.uk/maps/place/Close+To+The+Edge/@53.6927199,-2.2040092,266m/data=!3m1!1e3!4m2!3m1!1s0x0000000000000000:0xf00c8a3791f43fb8). Literally